# The code in wxterminal depends on gobject.
# The check for it is missing resulting in the link lacking -lgobject-2.0

diff -ru gnuplot-5.2.8.original/configure.ac gnuplot-5.2.8.patched/configure.ac
--- gnuplot-5.2.8.original/configure.ac	2019-11-27 22:52:29.000000000 +0000
+++ gnuplot-5.2.8.patched/configure.ac	2020-08-12 16:35:34.688947535 +0000
@@ -1071,7 +1071,7 @@
 if test "${with_cairo}" = yes ; then
   dnl cairo terminals
   PKG_CHECK_MODULES_NOFAIL(CAIROPDF,dnl
-       [cairo >= 1.2 cairo-pdf >= 1.2 pango >= 1.22 pangocairo >= 1.10 glib-2.0 >= 2.28])
+       [cairo >= 1.2 cairo-pdf >= 1.2 pango >= 1.22 pangocairo >= 1.10 glib-2.0 >= 2.28 gobject-2.0 >= 2.28])
   if test $pkg_failed != no; then
     AC_MSG_WARN([The cairo terminals will not be compiled.])
     with_cairo=no
@@ -1392,7 +1392,7 @@
 if test "$with_cairo" = yes; then
   AC_MSG_RESULT([  cairo-based pdf and png terminals: yes ])
 else
-  AC_MSG_RESULT([  cairo-based terminals: no (requires cairo>=1.2, pango>=1.22, glib>=2.28)])
+  AC_MSG_RESULT([  cairo-based terminals: no (requires cairo>=1.2, pango>=1.22, glib>=2.28 including gobject >= 2.28)])
 fi
 
 if test "$with_lua" = yes; then
